packages/examples/bike_simulator.py

```diff
@@ -38,9 +38,14 @@
 
 class PackageClass(LiotaPackage):
 
-    def run(self, registry):
+    def run(self, registry, package_record):
         from liota.entities.devices.bike_simulated import BikeSimulated
         import pint
+        from liota.lib.utilities.utility import check_integrity
+
+        # verify file integrity first
+        if (check_integrity(package_record, __file__) == False):
```

Comment:
Reply to secursive:

No, that's not as you thought. When you pass in the loading cmd, you just provide the package name, NO file extensions. It is the Package manager who will do sanity check and decide which file exactly need to be loaded and fetched. Also please refer to my explanation/examples to Piyush.